spec/controllers/ops_controller/ops_rbac_spec.rb

```diff
@@ -344,8 +344,7 @@
 
   context "::MiqGroup" do
     before do
-      MiqGroup.seed
-      MiqRegion.seed
+      EvmDatabase.seed_primordial
```

Comment:
I suppose not, but this is another example of why I dislike the current seed setup. If I just put `MiqUserRole.seed` here (that's what is needed), it's still another record of 'what needs to be loaded in order' that's separate from other logic. IE, the next time something changes it will have to change here again, too.